src/backend/aspen/app/views/sample.py

```diff
@@ -283,12 +283,21 @@ def create_sample():
         )
         request_data = request.get_json()
 
+        duplicates_in_request: Union[
+            None, Mapping[str, list[str]]
+        ] = api_utils.check_duplicate_data_in_request(request_data)
+        if duplicates_in_request:
+            return Response(
+                f"Error processing data, either duplicate private_identifiers: {duplicates_in_request['duplicate_private_ids']} or duplicate public identifiers: {duplicates_in_request['duplicate_public_ids']} are trying to be uploaded at the same time, please remove duplicates before proceeding with upload.",
```

Comment:
"are trying to be uploaded at the same time" replaced with "exist in the upload files" may be slightly less ambiguous